analysis_service/base/migrations/0001_initial.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
I created [bug 1283187](https://bugzilla.mozilla.org/show_bug.cgi?id=1283187) to cover the followup work required.